packages/cozy-harvest-lib/src/models/ConnectionFlow.js

```diff
@@ -385,17 +435,21 @@ export class ConnectionFlow {
   async sendTwoFACode(code) {
     this.setState({ status: RUNNING_TWOFA })
     try {
+      // @ts-ignore
       logger.debug(`ConnectionFlow: Sending two fa code ${code}`)
       await this.saveAccount(accounts.updateTwoFaCode(this.account, code))
     } catch (error) {
       // eslint-disable-next-line no-console
+      // @ts-ignore
       logger.error(error)
       this.setState({ status: ERRORED, error })
     }
   }
 
   handleLoginSuccess() {
-    this.jobWatcher.handleLoginSuccess()
+    if (this.jobWatcher?.handleLoginSuccess) {
```

Comment:
why this test?

Reply:
I did the comment before the push of the code sorry. This test is now removed (I juste needed to fix some unit tests)